plugin/storage/es/spanstore/dbmodel/model.go

```diff
@@ -0,0 +1,77 @@
+package dbmodel
+
+import (
+	"github.com/uber/jaeger/model"
+)
+
+
+// ReferenceType is the reference type of one span to another
+type ReferenceType string
+
+// TraceID is a trace id found in a span
+type TraceID string
+
+// SpanID is the id of a span
+type SpanID uint64
+
+const (
+	// ChildOf means a span is the child of another span
+	ChildOf ReferenceType = "CHILD_OF"
+	// FollowsFrom means a span follows from another span
+	FollowsFrom ReferenceType = "FOLLOWS_FROM"
+)
+
+// Trace is a list of spans
+type Trace struct {
+	Spans []Span `json:"spans"`
+}
+
+// Span is a span denoting a piece of work in some infrastructure
+type Span struct {
+	TraceID       TraceID     `json:"traceID"`
+	ParentSpanID  SpanID	  `json:"parentSpanID"`
+	SpanID        SpanID      `json:"spanID"`
+	Flags         uint32      `json:"flags"`
+	OperationName string      `json:"operationName"`
+	References    []Reference `json:"references"`
+	Timestamp     uint64      `json:"timestamp"`
+	Duration      uint64      `json:"duration"`
+	Tags          []Tag   	  `json:"tags"`
+	Logs          []Log       `json:"logs"`
+	Process       Process     `json:"process"`
+}
```

Comment:
@yurishkuro I didn't notice before, and refactored everything to fit the preexisting UI model, since ES uses JSON. The latest commit reflects this change.